### PR TITLE
Dt 090417 user dashboard dev

### PR DESCRIPTION
**Pull request by DesTodo**
@emcooper LOL, I uploaded the picture as a joke and then I forgot. I imagine we're not gonna keep the made up accounts for the end product; I will try to replace it online, to test the edit.

### CODE REVIEW

File: app/controllers/dashboard_controller.rb [hunk 1]
```diff
@@ -1,7 +1,10 @@
 class DashboardController < ApplicationController
   before_action :require_default
-  
-  def show
 
+  def show
+    @favorited_companies = current_user.favorite_companies
+    @favorite_members = current_user.favorite_users
+    @companies = current_user.companies
+    @feedback = @companies.questions(current_user.companies)
```
Comment (emcooper) [inline app/controllers/dashboard_controller.rb:8]: might want to use a presenter here?

File: spec/features/user_visits_dashboard_spec.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+require 'rspec_helper'
+
+feature "user visits dashboard" do
+  scenario "it can see its favorite users" do
+    user_1, user_2, user_3 = create_list(:user, 3)
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user_1)
+
+    co_1, co_2 = create_list(:company, 2)
+  end
```
Comment (emcooper) [inline spec/features/user_visits_dashboard_spec.rb:9]: Did this test not get finished?

**Review by emcooper — CHANGES_REQUESTED** [review]

File: app/controllers/dashboard_controller.rb (hunk 1 above)
Comment (DesTodo) [reply app/controllers/dashboard_controller.rb:8]: That's a great idea. I'll keep it as a refactor.

File: spec/features/user_visits_dashboard_spec.rb (hunk 2 above)
Comment (DesTodo) [reply spec/features/user_visits_dashboard_spec.rb:9]: There is a bug here. I'll PR again tomorrow. I'll close it for now.

**Pull request closed without merging**